### PR TITLE
[USING] Wallet Load

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch adds a standalone chapter on how to load wallets.

### CODE REVIEW

File: docs/using-wasabi/WalletLoad.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+## Wallet Explorer
+
+On the right side of Wasabi is the `Wallet Explorer`, where you see an alphabetically sorted list of all the previously [generated](/WalletGeneration.md) wallets.
+
+![](/WalletExplorerUnloaded.png)
+
+You can load a wallet simply by double clicking the line.
+A coin icon will appear to indicate the loaded wallets.
```
Comment (yahiheb) [inline docs/using-wasabi/WalletLoad.md:23]:
```suggestion
A coin will appear on the wallet icon to indicate the loaded wallets.
```

File: docs/using-wasabi/WalletLoad.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+---
+
+## Wallet Explorer
+
+On the right side of Wasabi is the `Wallet Explorer`, where you see an alphabetically sorted list of all the previously [generated](/WalletGeneration.md) wallets.
+
+![](/WalletExplorerUnloaded.png)
+
+You can load a wallet simply by double clicking the line.
```
Comment (yahiheb) [inline docs/using-wasabi/WalletLoad.md:22]:
```suggestion
You can load a wallet simply by double clicking on it or by right-clicking on it and then choosing `Load Wallet`.
```

File: docs/using-wasabi/WalletLoad.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+There are two ways of loading your wallets in Wasabi, and you can load multiple wallets at the same time.
+
+[[toc]]
+
+---
+
+## Wallet Explorer
+
+On the right side of Wasabi is the `Wallet Explorer`, where you see an alphabetically sorted list of all the previously [generated](/WalletGeneration.md) wallets.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/WalletLoad.md:18]:
```suggestion
On the right side of Wasabi is the `Wallet Explorer`, where you see an alphabetically sorted list of all the previously [generated](/using-wasabi/WalletGeneration.md) wallets.
```

File: docs/using-wasabi/WalletLoad.md (hunk 1 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/WalletLoad.md:23]: Yes, this is in the PR #702

**Comment by MaxHillebrand** [comment]
Actually... this chapter should explain the whole synchronization process. we currently only have [the faq](https://docs.wasabiwallet.io/FAQ/FAQ-UseWasabi.html#synchronization) for it...

I think I will do this in a follow up PR, so to not make this too cluttered.

**Pull request merged**